### PR TITLE
Frontend ping endpoint

### PR DESCRIPTION
**Pull request by cuahsimarko**
1. Install and test node locally https://docs.google.com/document/d/1RRJO_T3c5127uVHxjL498Smfuo78qVPEHF0Fvi6xJHQ/edit
2. cd hydroshare/hs_discover
3. npm run build
4. mkdir static/js
5. cp -R templates/hs_discover/js/** static/js
6. cd static/js
7. mv app.####.js app.js (use whatever integers are generated)
8. mv chunk-vendors.####.js chunk-vendors.js
9. cd ../../
10. docker-compose -f local-dev.yml up
11.  Wait until you see Mezzanine ASCII art
open a new terminal tab
12. docker exec -it hydroshare python manage.py collectstatic
accept the warning to overwrite files
13. Open developer tools or make the browser window small until Mauriel can help with the CSS
14. browse to http://localhost:8000/ping

### CODE REVIEW

**Comment by cuahsimarko** [comment]
image
Until Mauriel can assist with CSS, you must make the window small until you see the menu hamburger otherwise the test text output is not visible

**Comment by cuahsimarko** [comment]
I've been using these, people may want to adapt some variant for automation (local dev and beta prod)

```
npm run build
rm static/css/*.css
rm static/js/*.js
cp templates/hs_discover/css/app*.css static/css/app.css
cp templates/hs_discover/css/chunk-vendors*.css static/css/chunk-vendors.css
cp templates/hs_discover/js/app*.js static/js/app.js
cp templates/hs_discover/js/chunk-vendors*.js static/js/chunk-vendors.js
```

File: hs_discover/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,11624 @@
+{
```
Comment (sblack-usu) [inline hs_discover/package-lock.json:1]: This file is over 11000 lines long.  Are all of these packages necessary?  How was it generated?

Comment (cuahsimarko) [reply hs_discover/package-lock.json:1]: package-lock.json is automatically generated and is a standard npm file. It is best practice to commit to source code. https://docs.npmjs.com/configuring-npm/package-lock-json.html

File: hs_discover/src/App.vue [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<template>
+    <div id="app">
+        <router-view/>asdf
```
Comment (sblack-usu) [inline hs_discover/src/App.vue:3]: Do we need asdf in here?

File: hs_discover/package-lock.json (hunk 1 above)
Comment (sblack-usu) [reply hs_discover/package-lock.json:1]: that's quite the dependency tree generated from such a short package.json file.  I ran it locally and it generated similar sized file to convince myself.

**Pull request merged**